nevergrad/optimization/optimizerlib.py

```diff
@@ -1327,6 +1332,8 @@ def __init__(
         num = len(optimizers)
         self.optims: tp.List[base.Optimizer] = []
         sub_budget = None if budget is None else budget // num + (budget % num > 0)
+        first_optimizer = True
+        default_sub_workers = max(1, num_workers // len(optimizers))
```

Comment:
use the same formula as for budjet to avoid to redistribute to first only and to make the code easier.